lib/cdo/properties.rb

```diff
@@ -65,10 +65,10 @@ def fetch_metrics
   }
 end
 
+# Fetches project count from metrics and
 # Puts project count into format of XX million for display
-def round_project_count_to_million(project_count)
-  project_count = (project_count / 1_000_000).floor
-  return project_count
+def fetch_rounded_project_count_to_million
```

Comment:
nit: awkward grammar in naming.  I recommend calling it`fetch_project_count` and returning a hash that includes a key something like `rounded_millions` which is what callers will dereference.